autogluon/scheduler/reporter.py

```diff
@@ -174,15 +174,29 @@ def __init__(self, process, local_reporter, dist_reporter):
 
     def run(self):
         while self.process.is_alive():
+
+            # breaking communication if process raises exception
+            if self.process.exception is not None:
+                error, traceback = self.process.exception
+                self.local_reporter.terminate()
+                self.dist_reporter(done=True, traceback=traceback)
```

Comment:
Not sure if done=True is OK here. In the end, the process aborted. done=True I think implies it stopped normally and published a metric, no?

Reply:
In general I agree with you that `done=True` is a bit confusing. We can remove it. It should work without it also.